src/transport/GroupSession.h

```diff
@@ -38,6 +38,8 @@ class GroupSession : public Session
     const char * GetSessionTypeString() const override { return "secure"; };
 #endif
 
+    OperationalId GetPeerOperationalId() const override { return OperationalId::UndefinedOperationalId(); }
```

Comment:
This seems odd, since a group session _does_ in fact have a concept of "the node that sent the message", complete with node id, etc.  Why are we not returning that here?

But given that we are not, why is this API that only makes sense for SecureSession on Session at all?  Seems like it just sweeps the fact that the return value from the other sessions is nonsense under the rug....